x/dclauth/keeper/msg_server_propose_add_account.go

```diff
@@ -67,5 +67,10 @@ func (k msgServer) ProposeAddAccount(goCtx context.Context, msg *types.MsgPropos
 		k.SetAccountO(ctx, *account)
 	}
 
+	_, isFound := k.GetRejectedAccount(ctx, accAddr)
```

Comment:
Why should we remove rejected account immediately? Can't we remove it when we have a quorum of approvals?